keras_cv/layers/preprocessing/random_shear.py

```diff
@@ -148,3 +148,173 @@ def get_config(self):
             }
         )
         return config
+
+    def _augment(self, inputs):
```

Comment:
is there a specific reason that we need to override this?  Isn't this exactly what the default _augment does?

Reply:
in _augment fn , parameters of augment_bounding_box fn have been changed to include image as well to rescale the bounding boxes. In base augmentation class, augment_bounding_box takes only bbox and transformation.

Reply:
@pranavjadhav001 augment_bounding_box has now changed to augment_bounding_boxes(self, image, bounding_boxes, transformation=None). The _augment() in BaseImageAugmentationLayer has been updated to use the new format. you can remove this override.